### ISSUE TITLE
Can't run cointainer in Raspberry Pi 4

### ISSUE THREAD

**Issue by manusoftar** [butlerx/wetty#386]
I installed Docker + Portainer into my Debian 10 (Buster) on my RaspberryPI 4, then I pulled the butlerx/wetty image successfully and created a container. The container gets created but it fails on loading. On the logs I only see this line

standard_init_linux.go:228: exec user process caused: exec format error

I don't know it it is indeed a bug or if I am doing something wrong, but I don't know anywhere else to ask for some help.

Thanks.

**Comment by Joulinar** [comment]
Simply using the container will not work on RaspberryPI 4 as Wetty did not support ARM based system ootb. As far as I can see, they offer amd64 container only. https://hub.docker.com/r/wettyoss/wetty/tags

**Comment by butlerx** [comment]
Currently we dont build arm containers there is a pr to add support for arm #379 but it ran in to issues with long build times and yarn installs. 

To work on a pi youll need to clone the repo and build the container manually.

**Comment by Joulinar** [comment]
Theoretically this was working beginning of 2021 on a test I did https://dietpi.com/phpbb/viewtopic.php?p=30131#p30131

> To work on a pi youll need to clone the repo and build the container manually.

But most probably it will fail same way as it was for you guys. Isn't it?

```
yarn install v1.22.17
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://registry.yarnpkg.com/date-fns/-/date-fns-2.28.0.tgz: ESOCKETTIMEDOUT".
info If you think this is a bug, please open a bug report with the information provided in "/usr/src/app/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by jpspringall** [comment]
Hi @manusoftar 

I have literally just pulled the latest source of this and used buildx to create an arm image basically doing the same as #379 
For my RaspberryPi 3B
It's available here: [arm wetty docker hub](https://hub.docker.com/r/jpspringall/wetty)

As I have literally just done it, I've not added any documentation but it's identical to this code
Attached is my docker-compose, renamed to .txt as can't upload a .yml file
[docker-compose.txt](https://github.com/butlerx/wetty/files/8340721/docker-compose.txt)

Cheers

James

**Comment by butlerx** [comment]
Closed in #375
